src/js/components/StatBlockView.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import BaseStatsSection from './sections/BaseStatsSection.jsx';
```

Comment:
I've never needed to add the extension -- might just be a case of configuring your bundler (e.g. webpack)